operations/helm/charts/mimir-distributed/templates/ingester/ingester-statefulset.yaml

```diff
@@ -52,8 +52,17 @@ spec:
     metadata:
       labels:
         {{- include "mimir.podLabels" $args | nindent 8 }}
+        grafana.com/no-downscale: {{$rolloutZone.noDownscale}}
+        grafana.com/prepare-downscale: {{$rolloutZone.prepareDownscale}}
       annotations:
         {{- include "mimir.podAnnotations" $args | nindent 8 }}
+        {{- if $rolloutZone.prepareDownscale }}
+        grafana.com/prepare-downscale-http-path: ingester/prepare-shutdown
+        grafana.com/prepare-downscale-http-port: 80
+        {{- end-}}
+        {{- if $rolloutZone.downscaleLeader }}
+        grafana.com/rollout-downscale-leader: downscaleLeader
```

Comment:
The value of this should be the name of the leader statefulset. For example, the leader of the zone B statefulset should be `mimir-ingester-zone-a`.

Reply:
my mistake, i meant to add this as a variable.